src/_nebari/stages/kubernetes_services/__init__.py

```diff
@@ -180,6 +180,8 @@ class CondaStore(schema.Base):
     image_tag: str = constants.DEFAULT_CONDA_STORE_IMAGE_TAG
     default_namespace: str = "nebari-git"
     object_storage: str = "200Gi"
+    max_workers: int = 50
+    worker_resources: dict = {"requests": {"cpu": "1", "memory": "4Gi"}}
```

Comment:
@Adam-D-Lewis This are the minimum recommended resources as per conda-store [docs](https://conda.store/conda-store/references/faq#what-are-the-resource-requirements-for-conda-store-server). 

Below is the resources consumed on a build (scikit-learn, pandas) when there is no sizing is in place (in develop)

![Image](https://github.com/users/pt247/projects/1/assets/8033215/b205bbd0-7842-4153-bb5a-a6915dca144a)

As you can see that this uses about 7 GB. 
If we simply increate resource allocation to 7GB. Nebari CI test fails to deploy (timeout - typical of lack of memory).

Should the resource sizing be a separate ticket? KEDA scales on Queue so its behaviour will remain same.

Reply:
`{"cpu": "1", "memory": "4Gi"}` works in current CI.
If you assign more, it fails. My question is about deciding what is the appropriate size. And if it's okay to keep that decision as a part of this PR.